docs/devguide/installation.rst

```diff
@@ -44,6 +44,13 @@ Windows
 OS X
 ~~~~
 
+After cloning the repository then install yotta using the app and launch the app. You can find the latest release of yotta for OS X at https://github.com/ARMmbed/yotta_osx_installer/releases/ .
+
+When you are at the yotta terminal execute the following commands. 
+>>> yt target bbc-microbit-classic-gcc-nosd
+>>> yt up
+>>> yt make
+You should have a firmware.hex file that you can now upload to your microbit.
```

Comment:
Where is the firmware.hex file?  Maybe simpler to just say: "Now follow the instructions in README.md".